src/model/layer/cudnn_activation.cc

```diff
@@ -81,7 +81,7 @@ const Tensor CudnnActivation::Forward(int flag, const Tensor& input) {
     CUDNN_CHECK(cudnnActivationForward(
         ctx->cudnn_handle, this->acti_desc_, &alpha, this->desc_,
         inblock->data(), &beta, this->desc_, outblock->mutable_data()));
-  }, {input.block()}, {output.block()});
+  }, {input.block()}, {output.block()}, "cudnnActivationForward");
```

Comment:
I haven't modified operations in files under the layer dir before, so they may be not well buffered.

Reply:
In my opinion, the folder src/model/layer is depecracted because it is not using the autograd API, so it has minimal value to be updated at this stage.